src/views/HomeView.vue

```diff
@@ -131,4 +188,73 @@ form {
   margin-inline-end: var(--spacing-default);
   color: var(--color-headings);
 }
+
+.forms {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--spacing-default);
+  align-items: flex-start;
+  justify-content: space-between;
+  margin-block-end: 2rem;
+}
+
+input[name="skillsId_cb"]:not(:checked) {
+  display: none;
+}
+
+label[for="searchInput"] {
+  flex: 1;
+}
+
+.search-bar {
+  width: 100%;
+}
+
+.forms > :first-child {
+  width: min(100%, var(--section-width));
+}
+
+.forms > :last-child {
+  width: min(100%, 20rem);
+}
+
+form {
+  display: flex;
```

Comment:
heb je gecheckt dat dit niet de contactmoment formulieren verstoord?

Reply:
yes, de css is scoped